yin/rubysonar-master/pom.xml

```diff
@@ -78,6 +78,13 @@
             <version>1.2</version>
         </dependency>
 
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
Does the tests run on CLI? I couldn't find surefire.. maybe they are using something else

Reply:
they do run.